src/qibocal/protocols/characterization/resonator_spectroscopy.py

```diff
@@ -86,23 +108,38 @@ def load(cls, path):
 
 
 def _acquisition(
-    params: ResonatorSpectroscopyParameters, platform: Platform, qubits: Qubits
+    params: ResonatorSpectroscopyParameters,
+    platform: Platform,
+    qubits: Qubits,
 ) -> ResonatorSpectroscopyData:
-    """Data acquisition for resonator spectroscopy."""
+    """Data acquisition for resonator spectroscopy attenuation."""
```

Comment:
```suggestion
    """Data acquisition for resonator spectroscopy."""
```
The attenuation is already described in the parameters. I would call this just `resonator_spectroscopy` at this point.